### PR TITLE
Link to a list of important skinning guides and threads from Guides; update the Russian translation of Guides

### PR DESCRIPTION
**Pull request by TicClick**

### CODE REVIEW

File: wiki/Guides/ru.md [hunk 1]
```diff
@@ -1,48 +1,51 @@
 # Руководства
 
-Ниже приведен упорядоченный список руководств, созданный членами сообщества osu!. Большинство из них были созданы на форуме osu!, но впоследствии перенесены в osu!вики
+Здесь приведён список руководств за авторством разных игроков osu!. Большинство из этих статей были сперва опубликованы на форуме osu! в отдельных темах, а затем викифицированы.
```
Comment (UristMcKobold) [inline wiki/Guides/ru.md:3]:
```suggestion
Здесь приведён список руководств за авторством разных игроков osu!. Большинство из этих статей было сперва опубликовано в отдельных темах на форуме osu!, а затем викифицировано.
```
Большинство было. "в отдельных темах" пораньше, чтобы не сбивать фокус из-за восклицательного знака, плюс так читается более приятно.

**Review by UristMcKobold — CHANGES_REQUESTED** [review]

**Review by UristMcKobold — APPROVED** [review]
Russian part of the PR looks good to go.

**Comment by RockRoller01** [comment]
I forgot to mention this when I talked to you yesterday, but "How to make properly centered spinners" is in both places. The /Guides page links to the ported wiki article, while the "Guide and Important Threads" article links to the original forum post.
The link to the forum post should probably be removed and all 3 guides on /Guides should be moved to "Guide and Important Threads" as the information is currently spread in 2 different places.

**Comment by TicClick** [comment]
for now, I've made `Skinning/Guides and important threads` refer to the wiki pages where possible.

ideally -- as I see it -- this article should disappear in favor of `Guides#Skinning` acting as a starting point (with the project links moved to https://github.com/ppy/osu-wiki/blob/master/wiki/Projects/en.md, because some links are already there)

**Comment by RockRoller01** [comment]
Oh sorry, I have somehow overlooked the changes done to "Skinning/Guides and important threads", which are exactly what I said.

**Review by Joehuu — APPROVED** [review]

**Pull request merged**